XenAdmin/Dialogs/PropertiesDialog.cs

```diff
@@ -320,7 +320,7 @@ private void Build()
                 }
                 if (isHost || isPool)
                 {
-                    NRPEEditPage = new NRPEEditPage(isHost);
+                    NRPEEditPage = new NRPEEditPage();

```

Comment:
Strictly speaking this is not wrong, but because we have a dedicated method that sets the IXenObject on the panel, it is more tidy if this is examined within that method.